modules/security_hub/variables.tf

```
@@ -15,3 +21,12 @@ variable "region" {
   default     = "eu-west-1"
   description = "The name of the AWS region where SecurityHub will be enabled"
 }
+
+variable "sns_security_subscription" {
+  type = list(object({
+    sns_endpoint          = string
+    sns_endpoint_protocol = string
```

```suggestion
    endpoint = string
    protocol = string
```

No need to include an `sns` prefix if the variable is about SNS already.

Does it make sense to trim `endpoint_protocol` down to just `protocol` too?